translator/reader/utils.cpp

```diff
@@ -186,7 +186,8 @@ AtArray *ReadLocalMatrix(const UsdPrim &prim, const TimeSettings &time)
 static void getMaterialTargets(const UsdPrim &prim, std::string &shaderStr, std::string *dispStr = nullptr)
 {
 #if PXR_VERSION >= 2002
-    UsdShadeMaterial mat = UsdShadeMaterialBindingAPI(prim).ComputeBoundMaterial();
+    // We want to get the material assignment for the "full" purpose, which is meant for rendering
```

Comment:
is this code executed in the render delegate as well ? I am wondering if this change can/will affect the interactivity in the viewport, any idea ?